symforce/codegen/codegen.py

```diff
@@ -393,56 +394,23 @@ def generate_function(
         self.namespace = namespace
 
         template_data = dict(self.common_data(), spec=self)
```

Comment:
Having the config be able to modify `template_data` would be useful. For example, to expose utility methods that are specific to a given language: https://github.com/symforce-org/symforce/compare/main...ZiplineTeam:add-rust-code-generation#diff-655b6a9779a72c1b59d1990ad1581a08986c7676a68d1f0c72a06018797e8c65R454

Reply:
That link isn't working for me?

Reply:
Eh, strange. In any case, it just adds another module to the `template_data` dict like so:
```
import rust_util
template_data.update(rust_util=rust_util) # <- rust_util can be called from jinja now.
```
Giving different language CodegenConfig the ability to modify the data passed to jinja could be useful.